src/ai/backend/manager/models/vfolder.py

```diff
@@ -1587,6 +1548,74 @@ class Meta:
     items = graphene.List(VirtualFolder, required=True)
 
 
+async def delete_vfolders(
+    requested_vfolders: Sequence[VFolderDeletionInfo],
+    *,
+    storage_manager: StorageSessionManager,
+    db: ExtendedAsyncSAEngine,
+    reporter: ProgressReporter | None = None,
+) -> None:
+    async def _task(reporter: ProgressReporter | None):
+        row_deletion_infos: list[VFolderDeletionInfo] = []
+        failed_deletion: list[tuple[VFolderDeletionInfo, str]] = []
```

Comment:
Simillar situation here - the name `failed_deletion` itself can be considered to reader as a flag that indicates if the operation is failed or not.